frontend/src/components/sidebar/index.tsx

```diff
@@ -127,18 +139,44 @@ export default class Sidebar extends Component<IProps, IState> {
           <Form.Group controlId="displayOptionsPreSucc">
             <Form.Label>Which papers should be displayed:</Form.Label>
             <Form.Check
-              type="checkbox"
-              id="checkbox-display-preceding-papers"
+              type="radio"
+              custom={true}
+              id="radio-display-preceding-papers"
               label="Preceding papers"
+              name="graphContent"
+              value={GraphContent.PRECEDING}
+              checked={this.state.currentGraphContent === GraphContent.PRECEDING}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                this.setState({currentGraphContent: GraphContent.PRECEDING});
+                this.props.onGraphContentChanged(GraphContent[e.currentTarget.value as keyof typeof GraphContent]);
+              }}
             />
             <Form.Check
-              type="checkbox"
-              id="checkbox-display-succeeding-papers"
+              type="radio"
+              custom={true}
```

Comment:
makes it look a bit nicer (replaces the Browser's default radio buttons with bootstrap radio buttons)